gradle/anko-asynchronous-tasks/app/src/main/java/com/irontec/kotlintest/models/WeatherObject.kt

```diff
@@ -0,0 +1,24 @@
+package com.irontec.kotlintest.models
+
+/**
+ * Created by axier on 24/4/15.
+ */
+public class WeatherObject {
+
+    public var main: String = ""
+    public var description: String = ""
+    public var temp: Float = 0.0f
+    public var temp_Max: Float = 0.0f
```

Comment:
Colton uses `camelCase` naming.
You can use [`@SerializedName`](http://google-gson.googlecode.com/svn-history/r432/trunk/gson/docs/javadocs/com/google/gson/annotations/SerializedName.html) annotation to specify a property name in JSON.